categories.xml

```diff
@@ -1,4 +1,5 @@
 <categories>
-	<category name="Uncategorized" slug="uncategorized"/>
+	<category name="Uncategorized" slug="uncategorized"></category>
+        <category name="Configuration and Utilities" slug="config"></category>
```

Comment:
Anything related to configuration would be in Configuration, and utility methods would be in Utilities.  Though I think you're best bet is to make a list of what will be documented, then create categories based on the content.